### ISSUE TITLE
1.4 Node Sass binding error.

### ISSUE THREAD

**Issue by AndreasGalster** [fourseven/meteor-scss#191]
I don't think that can be fixed on our side. This probably has to be changed within the package?
```

=> Errors prevented startup:

   While loading plugin `compileScssBatch` from package `fourseven:scss`:
   C:\Users\Andreas
   Galster\AppData\Local\.meteor\packages\fourseven_scss\3.8.0_1\plugin.compileScssBatch.os.windows.x86_32\npm\node_modu
les\meteor\compileScssBatch\node_modules\node-sass\lib\index.js:15:11:
   Missing binding C:\Users\Andreas
   Galster\AppData\Local\.meteor\packages\fourseven_scss\3.8.0_1\plugin.compileScssBatch.os.windows.x86_32\npm\node_modu
les\meteor\compileScssBatch\node_modules\node-sass\vendor\win32-ia32-46\binding.node
   Node Sass could not find a binding for your current environment: Windows 32-bit with Node.js 4.x

   Found bindings for the following environments:
   - Windows 32-bit with Node 0.10.x
   - Windows 64-bit with Node.js 4.x

   This usually happens because your environment has changed since running `npm install`.
   Run `npm rebuild node-sass` to build the binding for your current environment.
   at Object.<anonymous> (C:\Users\Andreas
   Galster\AppData\Local\.meteor\packages\fourseven_scss\3.8.0_1\plugin.compileScssBatch.os.windows.x86_32\npm\node_modu
les\meteor\compileScssBatch\node_modules\node-sass\lib\index.js:15:11)
   at Module._compile (module.js:409:26)
   at Object.Module._extensions..js (module.js:416:10)
   at Module.load (module.js:343:32)
   at Module.Mp.load (C:\Users\Andreas
   Galster\AppData\Local\.meteor\packages\meteor-tool\1.4.0-beta.10\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rei
fy\node\runtime.js:16:23)
   at Function.Module._load (module.js:300:12)
   at Module.require (module.js:353:17)
   at require (internal/module.js:12:17)
   at Object.<anonymous> (C:\tools\isobuild\bundler.js:1565:22)
   at Object.require (C:\tools\isobuild\bundler.js:1542:11)
   at meteorInstall.node_modules.meteor.compileScssBatch.plugin.compile-scss.js
   (packages/compileScssBatch/plugin/compile-scss.js:3:18)
   at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
   at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
   at <runJavaScript-25>:349:1
   at <runJavaScript-25>:355:3


=> Your application has errors. Waiting for file change.

```

**Comment by sebakerckhof** [comment]
https://github.com/meteor/meteor/issues/7297

**Comment by sebakerckhof** [comment]
Fixed in Meteor `1.4-beta.10`, clean your `$HOME/.meteor` and/or `.meteor/local `

**Comment by AndreasGalster** [comment]
Thanks, will try that. Currently been deleting for 30 minutes, 1 million files deleted so far ... O_o :D
